backend/app/services/transcription.py

```
@@ -37,28 +37,66 @@ def transcribe_audio(audio_path: str, model_name: str, language: str) -> dict:
+        if not os.path.exists(audio_path):
+            raise RuntimeError(f"Audio file does not exist: {audio_path}")
+
+        file_size = os.path.getsize(audio_path)
+        if file_size < 1000:
+            raise RuntimeError(f"Audio file is too small ({file_size} bytes), cannot transcribe")
+
         model = get_model(model_name)
         result = model.transcribe(audio_path, language=language)
         logger.info("Transcription completed successfully.")
         return result
     except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)